{
  "modified": "2024-06-25T13:41:07Z",
  "published": "2024-06-25T13:41:07Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-5829",
  "aliases": [
    "SNYK-PYTHON-PYTHONSQLITEV1-5932317"
  ],
  "summary": "Malicious code in pythonsqlitev1 (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "pythonsqlitev1",
        "purl": "pkg:pypi/pythonsqlitev1"
      },
      "versions": [
        "1.1.0"
      ]
    }
  ],
  "references": [
    {
      "type": "ARTICLE",
      "url": "https://medium.com/checkmarx-security/the-evolutionary-tale-of-a-persistent-python-threat-d19196d4ed15"
    },
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-PYTHONSQLITEV1-5932317"
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2024-04627",
        "import_time": "2024-06-28T02:50:22.655099118Z",
        "modified_time": "2024-06-25T13:41:07Z",
        "sha256": "068d973aaf0e7f23eec79fd55f691995b636f78d9cdd4290145a5e0ea64ce651",
        "source": "reversing-labs",
        "versions": [
          "1.1.0"
        ]
      }
    ]
  }
}
